xcode/Subconscious/Shared/Library/Func.swift

```diff
@@ -77,8 +77,8 @@ struct Func {
                     powf(2.0, Float(attempts))
                 )
             )
-            sleep(seconds)
-            
+            try await Task.sleep(for: .seconds(seconds))
+
```

Comment:
Not related, but I realised this was causing blocking issues while waiting for the gateway to provision.

Reply:
Namely, it was causing this issue https://github.com/subconsciousnetwork/subconscious/issues/710

Reply:
😅